### PR TITLE
my solution for div salad challenge

### PR DESCRIPTION
**Pull request by emyjamalian**
How to test: 
Verify my changes make sense and nothing is broken and there's no div any more

### CODE REVIEW

File: html-and-web/html-and-the-web_div-salad/index.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        <ul>
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>React</li>
+          <li>Painting rainbows</li>
+          <li>Party</li>
+        </ul>
+      </section>
+    </section>
```
Comment (jameslaw195) [inline html-and-web/html-and-the-web_div-salad/index.html:69]: is this an extra tag?

**Review by jameslaw195 — COMMENTED** [review]
looks great!

**Review by jameslaw195 — APPROVED** [review]
Everything looks great.

File: html-and-web/html-and-the-web_div-salad/index.html (hunk 1 above)
Comment (emyjamalian) [reply html-and-web/html-and-the-web_div-salad/index.html]: Good catch. This was supposed to be main. resolved here [69e148f](https://github.com/emyjamalian/challenges/pull/1/commits/69e148f950a359d0276a6141a6dfa82acf5ee6a3)

**Pull request merged**